src/lightning/app/cli/commands/rm.py

```diff
@@ -0,0 +1,104 @@
+# Copyright The Lightning AI team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import click
+import lightning_cloud
+import rich
+
+from lightning.app.cli.commands.ls import _add_colors, _get_prefix
+from lightning.app.cli.commands.pwd import _pwd
+from lightning.app.utilities.app_helpers import Logger
+from lightning.app.utilities.cli_helpers import _error_and_exit
+from lightning.app.utilities.network import LightningClient
+
+logger = Logger(__name__)
+
+
+@click.argument("rm_path", required=True)
+@click.option("-r", required=False, hidden=True)
+@click.option("--recursive", required=False, hidden=True)
+def rm(rm_path: str, r: bool = False, recursive: bool = False) -> None:
+    """Delete files on the Lightning Cloud filesystem."""
+
+    root = _pwd()
+
+    if rm_path in [".", ".."]:
+        return _error_and_exit('rm "." and ".." may not be removed')
+
+    if ".." in rm_path:
+        return _error_and_exit('rm ".." or higher may not be removed')
+
+    root = os.path.join(root, rm_path)
+    splits = [split for split in root.split("/") if split != ""]
+
+    if root == "/" or len(splits) == 1:
+        return _error_and_exit("rm at the project level isn't supported")
```

Comment:
PR title says `Add rm at the project level` but this says it's not supported

Reply:
<img width="788" alt="Screenshot 2023-02-14 at 11 05 47" src="https://user-images.githubusercontent.com/12861981/218718819-203a9dd5-abe1-4cd3-851a-0e4510f5265e.png">

Renamed ;)